sevntu-checks/src/test/java/com/github/sevntu/checkstyle/checks/coding/PreferMethodReferenceCheckTest.java

```diff
@@ -0,0 +1,159 @@
+////////////////////////////////////////////////////////////////////////////////
+// checkstyle: Checks Java source code for adherence to a set of rules.
+// Copyright (C) 2001-2021 the original author or authors.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+////////////////////////////////////////////////////////////////////////////////
+
+package com.github.sevntu.checkstyle.checks.coding;
+
+import static com.github.sevntu.checkstyle.checks.coding.PreferMethodReferenceCheck.MSG_METHOD_REF;
+import static com.google.common.truth.Truth.assertThat;
+
+import org.junit.jupiter.api.Test;
```

Comment:
https://github.com/sevntu-checkstyle/sevntu.checkstyle/blob/e26546952caf1bec4688bb219562db15120b1dfc/sevntu-checks/src/test/java/com/github/sevntu/checkstyle/checks/coding/AvoidConstantAsFirstOperandInConditionCheckTest.java#L24

Try to use old junit

Reply:
Thanks, it solved the issue. But why was this happening?

Reply:
Junit5 requiring few more config, sevntu use old junit, so methods with new annotations are recognized by old junit

Reply:
Ok. Thanks for letting me know.